mkdocs.yml

```diff
@@ -141,6 +141,7 @@ nav:
           - How-To Guides:
               - Configuring PTP: roles/eos_designs/docs/how-to/ptp.md
               - Custom Structured Configuration: roles/eos_designs/docs/how-to/custom-structured-configuration.md
+              - Integrating AVD with CVaaS: roles/eos_designs/docs/how-to/avd-to-cvaas.md
```

Comment:
I don't think this is the right place for this.
Let's add it under the role `eos_config_deploy_cvp`. I cannot add it in a suggestion here, but change the config below like this:
```yaml
        - eos_config_deploy_cvp:
            - Overview: roles/eos_config_deploy_cvp/README.md
            - Integrating AVD with CVaaS: roles/eos_config_deploy_cvp/docs/avd-to-cvaas.md
```
And move the file to that new location.

Reply:
Sounds good, I'll move it.